.changeset/hot-kings-write.md

```diff
@@ -0,0 +1,18 @@
+---
+"@fluidframework/aqueduct": major
+"@fluidframework/container-runtime": major
+"@fluidframework/data-object-base": major
+"@fluidframework/datastore": major
+---
+
+`initializeEntryPoint` will become required
+
+The optional `initializeEntryPoint` method has been added to a number of constructors. This method argument will become required and a value will need to be provided to the following classes:
```

Comment:
You should that this will replace the request pattern and add a link to the doc @markfields talked about [here](https://github.com/microsoft/FluidFramework/pull/16506#discussion_r1280741888).